test/functional/helpers/dom/addAnchorToBody.js

```diff
@@ -1,14 +1,24 @@
 import { ClientFunction } from "testcafe";
 
 const addAnchorToBody = ClientFunction(cfg => {
-  const anchor = document.createElement("a");
-  const keys = Object.keys(cfg.attributes);
-  keys.forEach(key => {
-    anchor.setAttribute(key, cfg.attributes[key]);
-  });
-  const anchorText = document.createTextNode(cfg.text);
-  anchor.appendChild(anchorText);
-  document.body.appendChild(anchor);
+  const buildNode = ({ element = "a", attributes = {}, children, text }) => {
```

Comment:
We also have createNode in src/utils/dom/ so if we can re-use that it might make it a bit cleaner.